app/controllers/api/v1/dynamic_field_groups_controller.rb

```diff
@@ -46,15 +50,23 @@ def destroy
 
         def create_params
           params.require(:dynamic_field_group).permit(
-            :string_key, :display_label, :sort_order, :is_repeatable, :xml_translation, :parent_type, :parent_id
+            :string_key, :display_label, :sort_order, :is_repeatable, :parent_type, :parent_id
           )
         end
 
         def update_params
           params.require(:dynamic_field_group).permit(
-            :display_label, :is_repeatable, :xml_translation, :sort_order, :parent_type, :parent_id
+            :display_label, :is_repeatable, :sort_order, :parent_type, :parent_id
           )
         end
+
+        def export_rules_attributes
```

Comment:
I like your solution for making used of Rails nested attributes even when the public API doesn't expose the word "attributes", but I feel a little weird about having this method called `export_rules_attributes` since that ties the extraction of the param data to a Rails concept that doesn't have to do with param parsing.  Do you mind if we rename the method to `export_rules`, but still use that method in the way you're currently using it in the create and update methods?

Reply:
I like the current name because it makes it really clear were the parameters are being set on the instance of dynamic_fields. I wouldn't want to change it to `export_rules` because that makes it seem like its returning an array of export_rule objects. We can change it to `export_rule_params` if that's better.

Reply:
I merged both of this methods, after our discussion on chat